docker/jenkins/unit.sh

```diff
@@ -23,10 +23,8 @@ function cleanup {
     docker-compose -f $COMPOSE_FILE_LOC \
                    -p $COMPOSE_PROJECT_NAME \
                    down --remove-orphans
-    docker ps -a --no-trunc  | grep $COMPOSE_PROJECT_NAME \
-        | awk '{print $1}' | xargs --no-run-if-empty docker stop
-    docker ps -a --no-trunc  | grep $COMPOSE_PROJECT_NAME \
-        | awk '{print $1}' | xargs --no-run-if-empty docker rm
+    # Untag LB images that were built before this test run
```

Comment:
yeah, I specifically added "Untag", because I understand that if we have an image with 10 layers, `docker image rm` will only remove the image tag from the most recent layer, and not delete other layers if they're also used by other images. I will double-check this assumption and update the text if needed.
For simplicity's sake, perhaps "remove" is clearer.